src/deserialize_sequence.cc

```diff
@@ -142,6 +142,36 @@ void extract_time_of_last_execution(gul14::string_view extract, Step& step)
 
 } // anonymous namespace
 
+std::istream& operator>>(std::istream& stream, Sequence& sequence)
```

Comment:
Let me explain my reasoning again. Code like the following suggests that you can import an entire `Sequence` from a stream:
```cpp
Sequence seq;
seq << stream;
```
The impression is wrong because the stream operator does not import the steps at all. After using it, there is in fact no sequence, but only a few imported parameters. Therefore I suggest implementing this with a named function. `load_sequence_parameters()` describes the function well enough. If it should take a file or a stream as input could be discussed, but giving the function a name is much better than using the anonymous operator in this case.

Reply:
Dropping the commit that changed that.